GameEngine/robot_model.py

```diff
@@ -91,6 +92,14 @@ def get_position_components_wcs(self) -> (float, float):
         """
         return self._convert_EFCS_to_field_CS(self._x_pos_EFCS, self._y_pos_EFCS)
 
+    def get_point_angle_wcs(self):
+
+        return self.pointing_angle
+
+    def get_axis_len(self):
+
+        return self._axis_len
```

Comment:
all parameters with underscore before name (`_my_param`) should be internal, if you require it outside the object - rename it to `self.axis_len` (remove first underscore), get value directly (`my_object.axis_len`) and remove getter - it is not Java :p

Reply:
Gotcha